napari/layers/surface/normals.py

```diff
@@ -32,7 +32,7 @@ class Normals(EventedModel):
 
     mode: NormalMode = Field(NormalMode.FACE, allow_mutation=False)
     visible: bool = False
-    color: ColorValue = 'black'
+    color: ColorValueType = 'black'
```

Comment:
This is relying on the custom type validation described in the Pydantic docs: https://docs.pydantic.dev/latest/usage/types/#classes-with-__get_validators__

which I think is why some of the tests fail.

The assignment operator here should go through the validator (i.e. `ColorValue.validate`), but I'm guessing mypy can't infer that, which is understandable.

I'm considering a couple of alternatives here - will likely write a PR on top of this in the next day or two.